frontend/src/components/dashboard/Analysis.tsx

```diff
@@ -386,6 +454,44 @@ const Analysis = ({
           display: true,
           position: "right",
           align: "center",
+          labels: {
+            // hide labels starting with "_"
+            filter: (legendItem) => !legendItem.text.startsWith("_"),
+          },
+          onClick(_e, legendItem, legend) {
+            const legendIdx = legendItem.datasetIndex;
+
+            if (legendIdx === undefined) {
+              return;
+            }
+
+            const ci = legend.chart;
```

Comment:
is this an alias or do we need to save the reference?
Either way, I'd opt for a more descriptive variable name :)